lang/uk/request.php

```diff
@@ -33,13 +33,13 @@
     'claim-anon-choose'   => 'Будь ласка, робіть свій вибір уважно',
     'claim-as-anon'       => 'Чи хотіли б Ви створити заявку анонімно?',
     'claim-now'           => 'Створити заявку на виконання',
-    'claimed'             => 'Заявлено',
-    'claimed-success'     => 'Request Successfully Claimed',
+    'claimed'             => 'Створено',
+    'claimed-success'     => 'Запит успішно створено',
```

Comment:
Can you double check these are correct? Google translate tells me "The request was created successfully" while the translations are referring to claiming the request. Claiming a request is different than creating a request. Claiming is referring to the act of reserving the request so that no one else is allowed to fill it except for yourself. Claims automatically expire after 1 week.